_docker/drupal/drupal-filesystem/web/modules/custom/rhd_eloqua_embed/src/Controller/EloquaEmbedController.php

```diff
@@ -9,19 +9,32 @@
 use Drupal\assembly\Entity\Assembly;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Drupal\Core\Template\Attribute;
+use Drupal\Component\Utility\Html;
 
 class EloquaEmbedController {
 
   public function getForm(Assembly $assembly) {
     if($assembly->hasField('field_eloqua_json')) {
+      $options = $_GET['options'];
       $eloqua_json = $assembly->get('field_eloqua_json')->value;
       if(!$eloqua_json) {
         throw new NotFoundHttpException();
       }
+      $field_url_array = $assembly->get('field_url')->getValue();
+      $redirect_url = ($field_url_array[0]['uri']) ? $field_url_array[0]['uri'] : FALSE;
```

Comment:
This is a causing a regression.

`Notice: Undefined offset: 0 in Drupal\rhd_eloqua_embed\Controller\EloquaEmbedController->getForm() (line 26 of /var/www/drupal/web/modules/custom/rhd_eloqua_embed/src/Controller/EloquaEmbedController.php)`

I have created this Bug ticket to track it: https://issues.jboss.org/browse/DEVELOPER-5691

It is line 26 because Matt's more recent PR pushes line 25 down a line.

Reply:
* I guess not a regression, just a new error